docs/tracing/docs/010-details-jaeger.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Jaeger
+type: Details
+---
+
+## Overview
+[Jaeger](http://jaeger.readthedocs.io/en/latest/) is a monitoring and tracing tool for microservice-based distributed systems.
+It is used for monitoring and troubleshooting microservice-based distributed systems, including:
+
+- Distributed context propagation.
+- Distributed transaction monitoring.
+- Root cause analysis.
+- Service dependency analysis.
+- Performance and latency optimization.
+
+## Usage 
+
+The Envoy sidecar uses Jaeger to trace the request flow in the Istio service mesh. Jaeger is compatible with the Zipkin protocol, which Istio and Envoy use to communicate with the tracing back-end. This allows you to use the Zipkin protocol and clients in Istio, Envoy, and the Kyma services.
+
+For more details, see [Istio Distributed Tracing](https://istio.io/docs/tasks/telemetry/distributed-tracing.html).
+
+## Install Jaeger locally
+While Jager installs automatically during cluster installation, local Jaeger installation is optional. You can install Jaeger on a Kyma instance and run it locally using Helm.
+
+To install Jaeger locally, go to the `~/go/src/github.com/kyma-project/kyma/resources/` directory and run the following command:
```

Comment:
Is there some convention to provide the relative path? Should that be followed?

Reply:
Yes, it should be provided and this will be addressed and changed once we come up with a way to provide such paths.